modules/kernel/src/main/scala/tofu/concurrent/Atom.scala

```diff
@@ -58,25 +55,6 @@ object Atom {
     }
   }
 
-  final case class AtomByRef[F[_], A](ref: Ref[F, A]) extends Atom[F, A] {
```

Comment:
These changes were in #709, so I guess there is something wrong with branches 🤔

Reply:
These two are just following up PRs